src/app/main/component/user/components/profile/profile-widget/profile-header/profile-header.component.scss

```diff
@@ -11,9 +11,9 @@ p {
   position: relative;
 
   .edit-icon {
-    width: 20px;
-    height: 20px;
-    background-image: url('~/assets/img/profile/icons/edit-line.svg');
```

Comment:
can you please check if we are using this icon elsewhere
if not, please remove it )

Reply:
yes, we are using icon edit-line in one-habit.component.scss. Fix there too?

Reply:
no